data/tooltips.txt

```diff
@@ -155,6 +155,9 @@ tip "jump speed:"
 tip "jump fuel:"
 	`How much fuel is consumed when you jump between systems using this drive.`
 
+tip "maintenance costs:"
+	`How many credits per day you need to spend to maintain this outfit. Even if a ship is parked, you still need to pay for its maintenance.`
```

Comment:
Sorry, but I have another exception/request; can we have those maintenance costs also for outfits in cargo? (Who-ever implements planetary storage should also add it there, but planetary storage is currently not merged, so you don't need to consider that now.)

Reply:
Yeah, I was just thinking that after adding operating costs that stored outfits would eventually also need to have their maintenance accounted for, and by extension in cargo outfits should now as well.

Reply:
Alright, should have this done. Given that the player is only charged for crew salaries/maintenance costs while in space, I think that only checking the ship cargo is necessary, and the player's pooled cargo never needs checked.